lib/msf/core/module_manager.rb

```diff
@@ -27,13 +27,6 @@ class ModuleManager
 
     include Enumerable
 
-    #
-    # CONSTANTS
-    #
-
-    # Maps module type directory to its module type.
-    TYPE_BY_DIRECTORY = Msf::Modules::Loader::Base::DIRECTORY_BY_TYPE.invert
```

Comment:
Moved to the loader concern; I'm not strongly opinionated about this being removed / duplicated / kept here